lib/writable.js

```diff
@@ -24,3 +24,21 @@ ZWritable.prototype._read = function() {
 	this.push(null);
 };
 
+ZWritable.prototype.wrap = function(stream) {
```

Comment:
OK, but the pattern we've established is to create classes for this kind of thing; even if the methods are trivial (as in ThroughStream).